fiat-roles/src/main/java/com/netflix/spinnaker/fiat/roles/UserRolesSyncer.java

```diff
@@ -105,7 +105,7 @@ public void onApplicationEvent(RemoteStatusChangedEvent event) {
     isEnabled.set(isInService());
   }
 
-  @Scheduled(fixedDelay = 30000L)
+  @Scheduled(fixedDelayString = "${fiat.writeMode.syncDelayMs?:600000}")
```

Comment:
Not sure if the `?` will work here - would change to `syncDelayMs:600000`. I basically agree with @ajordens here, since polling for the lock every 30 seconds is at worst doing no harm.

Reply:
The `?` is needed otherwise the `RolesControllerSpec` fails with:
```
Caused by: java.lang.IllegalArgumentException: null
	at java.util.concurrent.ScheduledThreadPoolExecutor.scheduleWithFixedDelay(ScheduledThreadPoolExecutor.java:611)
```

Reply:
Silly Java not having the Elvis operator :-1: